index.js

```diff
@@ -157,6 +165,27 @@ class Conf {
 		this.store = plainObject();
 	}
 
+	_migrate(options) {
+		const runningVersion = this.store.__packageVersion__ || '0.0.0';
```

Comment:
You need to ensure the user doesn't accidentally set/change `__packageVersion__`, which cause weird problems.

Reply:
`__packageVersion__` should also not be available in `this.store`. It's an internal value.